client/app/scripts/superdesk-authoring/views/change-image.html

```diff
@@ -0,0 +1,32 @@
+<div class="modal-header">
+    <h3 class="pull-left">{{ activity.label|translate }}</h3>
+    <button type="button" class="close" ng-click="close()"><i class="icon-close-small"></i></button>
+</div>
+
+<div class="modal-body">
+    <div>
+        <label translate>Make sure you're looking the best</label>
+        <div><label class="cords-label">{{preview.cords}}</label></div>
+    </div>
+    <section class="main">
+        <div class="preview">
+            <div class="crop-area image" ng-hide="false">
+                <div class="original">
+                    <div sd-image-crop data-src="data.renditions.original.href" data-show-Min-Size-Error="true" data-cords="preview.cords" data-box-width="data.cropsize.width" data-box-height="data.cropsize.height" data-aspect-ratio="data.aspectR" data-minimum-size="[data.cropsize.width, data.cropsize.height]" data-crop-select="data.renditions[data.cropsize.name]"></div>
```

Comment:
Also, it might make sense to pass directive settings in a single argument, instead of having to provide numerous attributes with each holding only a single value.